cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -41,6 +43,7 @@
 import static com.linkedin.kafka.cruisecontrol.executor.ExecutionTask.State.ABORTING;
 import static com.linkedin.kafka.cruisecontrol.executor.ExecutionTask.State.DEAD;
 import static com.linkedin.kafka.cruisecontrol.executor.ExecutionTask.State.IN_PROGRESS;
+import static com.linkedin.kafka.cruisecontrol.executor.ExecutorState.State.*;
```

Comment:
Can we avoid wildcard import here?

Reply:
Maybe it's fine in this case because we are importing from `ExecutorState.State`, since `.State.*` only has the enums. This avoids having to use `ExecutorState.State.SOME_STATE` everywhere

Reply:
@efeg yes only `LEADER_MOVEMENT_TASK_IN_PROGRESS` is used, Alex's point is that this is an enum, there is no big overhead by using wildcard. But yes I feel it is more elegant not using wildcard.